scripts/check_pr_scope.py

```
@@ -0,0 +1,196 @@
+Usage:
+    python scripts/check_pr_scope.py [--branch <branch>] [--strict]
+"""
+
+import subprocess
+import sys
+
+
+def run_command(cmd: list[str]) -> tuple[int, str, str]:
+    """Run a command and return (returncode, stdout, stderr)."""
+    try:
+        result = subprocess.run(
```

Ruff (0.13.1)

`subprocess` call: check for execution of untrusted input

(S603)